src/app/components/banner/Banner.jsx

```diff
@@ -21,10 +18,12 @@ const Banner = ({ data, handleBannerSave }) => {
     if (data && !isEmptyObject(data) && !show) {
         return (
             <>
-                <div className={clsx("banner", "margin-top--1", data.type)}>
+                <div data-testid="banner" className={clsx("banner", "margin-top--1", data.type)}>
```

Comment:
Some `aria` controls are needed for users with visual impairments

Reply:
which exactly please. I used the generic html elements in form which do not need additional arias as they all have labels which are linked to the active elements. Banner is a text which will be read to user.

Reply:
If you have an `error summary` as the patterns suggest you should have. Then you can include `role="alert"` in the containing `div`